test/code_block_fenced.md

```diff
@@ -9,6 +9,12 @@ And here is more text
 This is a code block that won't trigger.
 ```
 
+```text
+This is code
+
```

Comment:
Can’t tell in GitHub PR: Are there spaces in the line?

Reply:
No, and now that I think of it, that test doesn't make sense. At first, I wanted to validate the behavior didn't change for fenced code blocks, but now I understand your test logic better. Let me update this.

Reply:
(Really like what you did with the tests, btw 👍 . Nice job)